cea/resources/radiation/geometry_generator.py

```
@@ -235,23 +367,29 @@ def print_progress(i, n, _, __):
         return False
 
 
 class BuildingGeometry(object):
-    __slots__ = ["name", "windows", "walls", "roofs", "footprint", "orientation_walls", "orientation_windows",
-                 "normals_windows", "normals_walls", "intersect_walls", "terrain_elevation"]
+    __slots__ = ["name", "terrain_elevation", "footprint",
+                 "windows",    "orientation_windows",    "normals_windows",    "intersect_windows",
+                 "walls",      "orientation_walls",      "normals_walls",      "intersect_walls",
+                 "roofs",      "orientation_roofs",      "normals_roofs",      "intersect_roofs",
+                 "undersides", "orientation_undersides", "normals_undersides", "intersect_undersides",
+                ]
```

Flake8 (7.2.0)

[error] closing bracket does not match visual indentation

(E124)